docs-source/content/userguide/experimental/istio.md

```diff
@@ -89,52 +79,85 @@ metadata:
     weblogic.domainUID: domain2
 spec:
   ... other content ...
-  experimental:
+  configuration:
     istio:
       enabled: true
       readinessPort: 8888
+      envoyPort: 31111
 ```
 
-To enable the experimental Istio support, you must include the `istio` section
+To enable the Istio support, you must include the `istio` section
 and you must set `enabled: true` as shown.  The `readniessPort` is optional
-and defaults to `8888` if not provided.  
+and defaults to `8888` if not provided.  The `envoyPort` is optional and defaults to `31111` if not provided.
 
 ##### How Istio-enabled domains differ from regular domains
 
-Istio enforces a number of requirements on pods.  When you enable Istio support, the
-domain on a persistent volume sample scripts will make the following adjustments
-to your domain in order to satisy Istio's requirements:
+Istio enforces a number of requirements on pods.  When you enable Istio support in the domain resource, the
+introspector job automatically creates configuration overrides with the necessary channels for the domain to satisfy Istio's requirements, including:
 
 * On the Administration Server:
-    * Create a channel called `istio-probe` with listen address `127.0.0.1:8888` (or
+    * A network channel called `istio-probe` with listen address `127.0.0.1:8888` (or
       the port you specified in the `readinessPort` setting).
-    * Create a channel called `istio-t3` with listen address `127.0.0.1` and the port
+    * A network channel called `istio-t3` with listen address `127.0.0.1` and the port
       you specified as the admin port.
-    * Create a channel called `istio-ldap` with listen address `127.0.0.1` and the port
+    * A channel called `istio-ldap` with listen address `127.0.0.1` and the port
       you specified as the admin port, with only the LDAP protocol enabled.
-    * Create a channel called `istio-T3Channel` with listen
-      address `127.0.0.1` and the port you specified as the T3 port.
+    * The introspector job does not create any configuration network channel for external access.  You can create a channel called `istio-T3Channel` with listen address `127.0.0.1` and the port you specified as the T3 port in your WebLogic domain configuration.
```

Comment:
There is nothing special about the "istio-T3Channel" name.

@jshum2479, we probably need to discuss this more.  Here's what I was intuitively expecting...  that the WebLogic default, default admin, and default secure channels (if enabled) would be replaced with a set of single-protocol channels with some naming pattern that avoids overlaps and then similarly for any configured NAP's.